public/js/makeTicket.js

```diff
@@ -147,3 +159,38 @@ function checkTicketDeadline(data) {
     throw Error("ticket deadline must be in the format YYYY-DD-MM");
   }
 }
+
+function checkTicketTags(data){
+    console.log(typeof data, data);
+    if (typeof data !== "string") throw `Error: ticket tags must be a string!`;
+    data = data.trim();
+    return data;
+}
+
+
+function checkIdArray (idArray, varName){
+    if (!idArray || !Array.isArray(idArray))
+      throw `You must provide an array of ${varName}`;
+    for (let i in idArray) {
+      if (typeof idArray[i] !== "string" || idArray[i].trim().length === 0) {
+        throw `One or more elements in ${varName} array is not a string or is an empty string`;
+      }
+      idArray[i] = idArray[i].trim();
+    }
+    if (!(idArray.length >= 1))
```

Comment:
While not explicitly an error, I'd say good convention is to put error checks for arrays before the array operation